### PR TITLE
Added Storage Templates

### PR DESCRIPTION
**Pull request by steven-tey**
### Type of Change

- [ ] New Example
- [ ] Example updates (Bug fixes, new features, etc.)
- [ ] Other (changes to the codebase, but not to examples)

### New Example Checklist

- [ ] 🛫 `npm run new-example` was used to create the example
- [ ] 📚 The template wasn't used but I carefuly read the [Adding a new example](https://github.com/vercel/examples#adding-a-new-example) steps and implemented them in the example
- [ ] 📱 Is it responsive? Are mobile and tablets considered?

### CODE REVIEW

File: storage/kv-redis-nuxt/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+npm run build
+```
+
+Locally preview production build:
+
+```bash
+npm run preview
+```
+
+Check out the [deployment documentation](https://nuxt.com/docs/getting-started/deployment) for more information.
```
Comment (lfades) [inline storage/kv-redis-nuxt/README.md:42]: Use the same readme structure that's used in other examples. If some things can't be added (like the deploy button, but it should work here) those can be omitted.

File: storage/kv-redis-nuxt/public/favicon.ico
Comment (lfades) [inline storage/kv-redis-nuxt/public/favicon.ico:1]: IMO the favicon should always be the Vercel logo.

File: storage/postgres-prisma/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+pnpm create next-app --example https://github.com/vercel/examples/tree/main/storage/postgres-prisma
+```
+
+Next, run Next.js in development mode:
+
+```bash
+pnpm dev
+```
+
+Deploy it to the cloud with [Vercel](https://vercel.com/new?utm_source=github&utm_medium=readme&utm_campaign=vercel-examples) ([Documentation](https://nextjs.org/docs/deployment)).
```
Comment (lfades) [inline storage/postgres-prisma/README.md:49]: This is missing the instructions for using the `.env.example` file that's being included. E.g: https://github.com/vercel/examples/tree/main/edge-middleware/feature-flag-split#clone-and-deploy

Comment (steven-tey) [reply storage/postgres-prisma/README.md:49]: Good point, will fix

File: storage/postgres-kysely/.env.example [hunk 3]
```diff
@@ -0,0 +1,6 @@
+POSTGRES_URL=
+POSTGRES_URL_NON_POOLING=
+POSTGRES_USER=
+POSTGRES_HOST=
+POSTGRES_PASSWORD=
+POSTGRES_DATABASE=
```
Comment (lfades) [inline storage/postgres-kysely/.env.example:6]: Should this variables also be part of the deploy button?

File: storage/postgres-prisma/README.md (hunk 2 above)
Comment (steven-tey) [reply storage/postgres-prisma/README.md:49]: @lfades PR is up: https://github.com/vercel/examples/pull/645

File: storage/postgres-kysely/.env.example (hunk 3 above)
Comment (steven-tey) [reply storage/postgres-kysely/.env.example:6]: no because we are using a special param for adding Vercel Postgres

**Pull request merged**